cloudfoundry.md

```diff
@@ -130,6 +130,8 @@ cf set-env foia DATABASE_URL [value]
 cf set-env foia FOIA_ANALYTICS_ID [value]
 cf set-env foia FOIA_SECRET_SESSION_KEY [value]
 cf set-env foia DJANGO_SETTINGS_MODULE foia_hub.settings.dev
+cf set-env foia NEW_RELIC_LICENSE_KEY=[value]
+cf set-env foia NEW_RELIC_APP_NAME=[value]
```

Comment:
The app name in the cloud foundry case is auto-determined by a script. I suggest using that script somehow to figure out the app name (at least in the cloud foundry case). 

Also, I don't know what the difference between foia, foia-a, and foia-b are. Should those be reflected in new relic? Is it confusing if the app changes that everytime?